packages/suite/src/components/suite/StakingProcess/InfoCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { Card, H3, Icon, Row } from '@trezor/components';
+import { spacings, spacingsPx } from '@trezor/theme';
+import { useState } from 'react';
+import styled from 'styled-components';
+
+const ContentWrapper = styled.div`
+    padding-top: ${spacingsPx.lg};
+`;
+
+interface InfoCardProps {
+    heading: React.ReactNode;
+    content: React.ReactNode;
+}
+
+export const InfoCard = ({ heading, content }: InfoCardProps) => {
```

Comment:
Please use CollapsibleBox for this. See stakeethinnushellmodal (or what's the name:)

Reply:
done https://github.com/trezor/trezor-suite/pull/14713/commits/43c8cfd532d265b225b284aa487560b4b30adf04